daffodil-schematron/src/test/scala/org/apache/daffodil/validation/ConvertSch.scala

```diff
@@ -0,0 +1,14 @@
+package org.apache.daffodil.validation
+//import org.scalatest.{Matchers, WordSpec}
+//
+//
+//class ConvertSch extends WordSpec with Matchers {
```

Comment:
Why commented out?

Reply:
Its a scalatest spec from the other repo.  Tests are still on the todo here...